packages/server/src/integrations/tests/sql.spec.ts

```diff
@@ -625,51 +228,6 @@ describe("SQL query builder", () => {
     })
   })
 
-  it("should lowercase the values for Oracle LIKE statements", () => {
-    let query = new Sql(SqlClient.ORACLE, limit)._query(
```

Comment:
We don't test oracle today anywhere else as its very difficult - this test case likely isn't covered.

Reply:
Good catch! Will reinstate this.